<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reviewWorkWindow</class>
 <widget class="QMainWindow" name="reviewWorkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="BodyLabel" name="BodyLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>当前项目：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BodyLabel" name="projectLabel">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="BodyLabel" name="BodyLabel_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>当前任务驳回原因：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BodyLabel" name="errorReasonLabel">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="PushButton" name="openProjectPb">
           <property name="text">
            <string>打开项目</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="refreshWorkPb">
           <property name="text">
            <string>刷新任务表(F5)</string>
           </property>
           <property name="shortcut">
            <string>F5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PillPushButton" name="loadWmsPb">
         <property name="text">
          <string>加载吉林一号在线底图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BodyLabel" name="processLabel">
         <property name="text">
          <string>1/???</string>
         </property>
         <property name="pixelFontSize" stdset="0">
          <number>18</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ListView" name="listView"/>
       </item>
       <item>
        <widget class="PushButton" name="giveUpPb">
         <property name="text">
          <string>废弃（慎点！！！）</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="RadioButton" name="rgbRender">
           <property name="text">
            <string>RGB渲染</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rgbRenderBtnGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="RadioButton" name="bgrRender">
           <property name="text">
            <string>BGR渲染</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rgbRenderBtnGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutRender">
         <item>
          <widget class="RadioButton" name="lineRender">
           <property name="text">
            <string>空心渲染</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">renderBtnGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="RadioButton" name="fillRender">
           <property name="text">
            <string>实心渲染</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">renderBtnGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="RadioButton" name="fillLineRender">
           <property name="text">
            <string>线条渲染</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">renderBtnGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PushButton" name="reSizeExtent">
         <property name="text">
          <string>重置大小</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Color">
         <item>
          <widget class="BodyLabel" name="BodyLabel_2">
           <property name="text">
            <string>修改编辑矢量颜色</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="ComboBox" name="rejectCBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="reasonLineEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>33</height>
            </size>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="placeholderText">
            <string>补充原因（可空）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PushButton" name="rejectPb">
         <property name="text">
          <string>驳回</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="2">
          <widget class="PushButton" name="lastPb">
           <property name="toolTip">
            <string>上一个(A)</string>
           </property>
           <property name="text">
            <string>上一个</string>
           </property>
           <property name="shortcut">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="PushButton" name="nextPb">
           <property name="toolTip">
            <string>下一个(D)</string>
           </property>
           <property name="text">
            <string>通过</string>
           </property>
           <property name="shortcut">
            <string>D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PillPushButton</class>
   <extends>ToggleButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../jixinRSDM/yoyiResource/icoRSAI.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="rgbRenderBtnGroup"/>
  <buttongroup name="renderBtnGroup"/>
 </buttongroups>
</ui>
